### ISSUE TITLE
Add support for videos

### ISSUE THREAD

**Issue by roborourke** [humanmade/aws-rekognition#3]
This is trickier to implement but doable:

- Requires file to be on S3
- Handled async via polling (start/get functions)

Thinking this is low priority for anyone as most video is handled by 3rd party providers.

**Comment by noeltock** [comment]
Like you say, quite low priority. Would be more about API completeness then business need/requirements (none today).

**Comment by joehoyle** [comment]
Yeah probably low - if we _did_ want to own Video, we'd also go into the AWS transcoding etc too. I'm sure we _could_ make a great experience there as we have a lot of AWS tooling at our fingertips but I don't think we've really seen demand for video.
